packages/react-core/src/components/ChipGroup/Chip.tsx

```diff
@@ -14,13 +14,13 @@ export interface ChipProps extends React.HTMLProps<HTMLDivElement> {
   closeBtnAriaLabel?: string;
   /** Additional classes added to the chip item */
   className?: string;
-  /** Flag indicating if the chip has overflow */
+  /** Flag indicating if the chip is an overflow chip */
   isOverflowChip?: boolean;
-  /** Flag if chip is read only */
+  /** Flag indicating if chip is read only */
   isReadOnly?: boolean;
-  /** Function that is called when clicking on the chip button */
+  /** Function that is called when clicking on the chip close button */
   onClick?: (event: React.MouseEvent) => void;
-  /** Internal flag for which component will be used for chip */
+  /** Component that will be used for chip. It is recommended that <button> or <li>  are used when the chip is an overflow chip. */
   component?: React.ReactNode;
```

Comment:
could this be more constrained to `'div' | 'li' | 'button'`?

Reply:
@mcoker and I talked about this yesterday and saw no need to constrain this.